client/web/compose/src/components/ModuleFields/Viewer/Record.vue

```diff
@@ -9,7 +9,6 @@
     <span
       v-for="(v, index) of formattedValue"
       :key="index"
-      class="text-nowrap"
```

Comment:
Am doubtful this won't impact other places where we use record viewer, the only changes should be in record organizer base